tests/Core/Button/FluentButtonTests.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.FluentUI.AspNetCore.Components.Tests.Button;
 
 public partial class FluentButtonTests : TestContext
 {
-    private TestContext TestContext => new(); // TODO: To remove and to use the `RenderComponent` inherited method.
+    private static TestContext TestContext => new(); // TODO: To remove and to use the `RenderComponent` inherited method.
```

Comment:
Why did you added `static`?

Reply:
Was a compiler suggestion

Reply:
Don't always believe VS ;-) I agree on some points, but using `static` can be dangerous in some cases. Personally, I prefer not to use it, even if it's more efficient at times.